library/ssl_tls.c

```diff
@@ -2883,7 +2883,7 @@ int mbedtls_ssl_decrypt_buf( mbedtls_ssl_context const *ssl,
      * Match record's CID with incoming CID.
      */
     if( rec->cid_len != transform->in_cid_len ||
-        mbedtls_platform_memcmp( rec->cid, transform->in_cid, rec->cid_len ) != 0 )
+        memcmp( rec->cid, transform->in_cid, rec->cid_len ) != 0 ) // use regular memcmp as CID is not that critical
```

Comment:
As in the other PR, I think it's better to say "public" than "not that critical" when it's true (which is is here).

Reply:
fixed